docs/guides/feature-flags.rst

```diff
@@ -26,4 +26,4 @@ Available Flags
 
 ``MKDOCS_THEME_RTD``: :featureflags:`MKDOCS_THEME_RTD`
 
-``DONT_SHALLOW_CLONE``: :featureflags:`DONT_SHALLOW_CLONE`
+``DONT_SHALLOW_CLONE``: :featureflags:`DONT_SHALLOW_CLONE` This flag is useful if your code accesses old commits during docs build, e.g. python-reno release notes manager is known to do that (error message line would probably include one of old Git commit id's).
```

Comment:
I'll put this as another paragraph, it doesn't look good with the autogenerated text from the `:featureflags:` role (the text doesn't end with a period).

Please follow https://docs.readthedocs.io/en/stable/docs.html#content to split lines.